### PR TITLE
Show problem resolved adjustments

### PR DESCRIPTION
**Pull request by llvasconcellos**
Adjustments following Fabian's code review

### CODE REVIEW

File: app/src/main/java/org/projectbuendia/client/diagnostics/Troubleshooter.java [hunk 1]
```diff
@@ -48,14 +48,14 @@ public Troubleshooter(EventBus eventBus) {
     }
 
     /**
-     * Returns true if the given issue is current active.
+     * Returns true iff the given issue is current active.
```
Comment (capnfabs) [inline app/src/main/java/org/projectbuendia/client/diagnostics/Troubleshooter.java:51]: I'm ok with you leaving this as `if`, it seems that `iff` isn't clear to a lot of people. Likewise, below.

File: app/src/main/java/org/projectbuendia/client/ui/BaseActivity.java [hunk 2]
```diff
@@ -335,66 +335,72 @@ public void onEventMainThread(TroubleshootingActionsChangedEvent event) {
...
-        troubleshootingMessages.put(HealthIssue.PACKAGE_SERVER_INDEX_NOT_FOUND, new Integer[] {
-            R.string.troubleshoot_package_server_misconfigured,
-            R.string.troubleshoot_package_server_misconfigured_solved,
-            10
-        });
-
-        Integer[] messages = troubleshootingMessages.get(solvedIssue);
+        // The troubleShootingMessages Map have the issue as the key and the TroubleshootingMessage
+        // object as it's value.
+        Map<HealthIssue, TroubleshootingMessage> troubleshootingMessages = new HashMap<>();
```
Comment (capnfabs) [inline app/src/main/java/org/projectbuendia/client/ui/BaseActivity.java:340]: This is much better. Thanks Leonardo!

**Comment by capnfabs** [comment]
Ok to merge. 👌

File: app/src/main/java/org/projectbuendia/client/diagnostics/Troubleshooter.java (hunk 1 above)
Comment (llvasconcellos) [inline app/src/main/java/org/projectbuendia/client/diagnostics/Troubleshooter.java:51]: If it isn't clear is something to learn. Now I know!

**Pull request merged**